CreeDictionary/API/views.py

```diff
@@ -1,14 +1,28 @@
-from django.http import JsonResponse
+from typing import List, Union
 
-# todo: update api documentation
+from API.schema import SerializedSearchResult
+from django.http import JsonResponse, HttpResponseBadRequest
+from .models import Wordform
 
 
-def translate_cree(request, query_string: str) -> JsonResponse:
+def click_in_text(request) -> Union[JsonResponse, HttpResponseBadRequest]:
```

Comment:
`HttpResponse` is the super class of both `JsonResponse` and `HttpResponseBadRequest`, so use that instead of `Union`! 👍🏼 

```suggestion
def click_in_text(request) -> HttpResponse:
```

Reply:
thanks so much I was wondering what base class I could use